index.bs

```diff
@@ -855,6 +856,10 @@ partial interface HTMLIFrameElement {
       resulted only in this report being generated (with no further action taken
       by the user agent in response to the violation).
 
+    - <dfn for="PermissionsPolicyViolationReportBody">allowAttribute</dfn>: If
+      known, the value of <{iframe}> element's <{iframe/allow}> attribute, or 
```

Comment:
In what cases would an attribute value not be known? Also, this does not refer to any specific iframe, just the concept of the <iframe> element in the HTML spec.

Reply:
`PermissionsPolicyViolationReportBody` is used in 2 violation types (i.e. permissions-policy-violation` and `potential-permissions-policy-violation`). And the `allowAttribute` is only set for `potential-permissions-policy-violation` type. Maybe I should say `if defined` insted of `if known`.

For refering to an iframe, what's the best way to indicate the iframe which is in question (in the spec wording)?

Reply:
Maybe something like this?

> For reports of potential violations, which can be attributed to a specific <{iframe}> element, the value of the <{iframe/allow}> attribute of that element, or omitted otherwise.